app/views/pages/home.html.erb

```diff
@@ -18,8 +18,55 @@
     </div>
     <% end %>
   </div>
+
```

Comment:
these changes don't seem to be related to the travel cost. is this part of @d-tello 's PR for the homepage edits?

Reply:
I was not changing something here ... so Im not sure about it ...